hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/state/Serializer.java

```diff
@@ -0,0 +1,28 @@
+package org.apache.mesos.hdfs.state;
```

Comment:
I don't believe this should live in hdfs package land... under mesos, probably a stream package.  I'm going to factor out common stuff after we are done here. so you can leave if you like.